src/ui/action.rs

```diff
@@ -527,6 +527,13 @@ pub fn set_as_wallpaper(sender: &Sender<Event>, file_list: &FileList) {
     }
 }
 
+pub fn copy_current_image(image_list: Rc<RefCell<ImageList>>) {
+    let display = gdk::Display::default().unwrap();
```

Comment:
You have to take sender as parameter in this function and handle the errors instead of unwrapping or using expect.
Using unwrap or expect will close the app when it occurs. The error should be displayed for the user and the app should continue running.
Please look at save_current_image function in this image to see how to handle them.

Reply:
Do I also need to handle errors when getting the Display? I think it's fair to assume that if the display server connection is dead, then something has went terribly wrong with the system, and the app probably can't continue to function anyway. It should only ever happen when X11/Wayland have crashed.